src/network/networkd-link.c

```diff
@@ -4420,6 +4421,13 @@ int link_save(Link *link) {
                 }
         }
 
+        r = sd_dhcp6_client_get_iaid(link->dhcp6_client, &iaid);
+        if (r >= 0) {
+                fputs("DHCP6_CLIENT_IAID=", f);
+                fprintf(f, "%u\n", iaid);
+                fputc('\n', f);
```

Comment:
Please merge them into single `fprintf()`.

For DHCPv4, IAID is printed with `0x%x`. I think it is better to use it for DHCPv6 too.

Reply:
yeah.